pages/roadmap.md

```diff
@@ -286,12 +220,82 @@ This table describes the biggest projects that we’re working on.
         </td>
     </tr>
     <tr>
-        <td><strong>Ktor</strong>
+        <td><strong>Website</strong></td>
+        <td>
+            <ul>
+                <li><a href="https://youtrack.jetbrains.com/issue/KT-44338" target="_blank">Revamp Kotlin documentation</a></li>
+                <li><a href="https://youtrack.jetbrains.com/issue/KT-44339" target="_blank">Make the Kotlin website mobile friendly</a></li>
+                <li><a href="https://youtrack.jetbrains.com/issue/KT-44340" target="_blank">Design a new Kotlin visual style</a></li>
+                <li><a href="https://youtrack.jetbrains.com/issue/KT-44342" target="_blank">Close <em>try.kotlinlang.org</em></a></li>
```

Comment:
Nit-picking: maybe another verb with more straightforward meaning: disable or shut down?

Reply:
It requires updating in 4 places including YT. I would keep it `Close`. Copyeditors are OK.